Code/GraphMol/RGroupDecomposition/RGroupDecompData.h

```diff
@@ -314,7 +314,8 @@ struct RGroupDecompData {
     }
 
     // Deal with non-user supplied labels
-    for (auto newLabel : indexLabels) {
+    for (auto it = indexLabels.begin(); it != indexLabels.end(); it++) {
```

Comment:
Why the change here?

And (I'm very guilty of this my self) ++it

Reply:
I think this can be reverted as you aren't changing the iterator order.

Reply:
I had put in a reverse iterator as I thought it could reduce the number of test failures (it didn't), then forgot to revert.

it++ - too much C and Java!

I have reverted in the latest commit.